docs/functions.rst

```diff
@@ -1607,6 +1607,7 @@ Arguments:
    `container`, Yes, `string`, name of the kanister sidecar container
    `includePath`, Yes, `string`, path of the data to be backed up
    `snapshotTags`, No, `string`, custom tags to be provided to the kopia snapshots
+   `repositoryServerUserHostname`, No, `string`, user's hostname to access the kopia repository server. Hostname would be available in the user access credential secret
```

Comment:
does second sentence mean, user should figure out the hostname by looking at the user access credential secret?
What does it mean?

Reply:
Yes ! User could provide the hostname which he/she wants to use by looking at user access credentials secret. If it's not provided we'll get the last entry from the user access map which is present inside the secret by default.